pkgvars

```diff
@@ -1,6 +1,6 @@
-pkgname=citus-ha
+pkgname=auto-failover-enterprise
```

Comment:
Is `pkgname` supposed to be `pg-auto-failover-enterprise` or `auto-failover-enterprise` as it is now?

Reply:
this seems indeed the case, however, the only place where this is used is a regex that changes the `postgres-XX-auto-failover-enterpise` package name to extend it with the major+minor version of the package. Since postgres is in the name there we have decided to not put `pg_` in the extension package.

Reply:
I see, thanks for explaining!